src/parser/jobs/sam/modules/Kaiten.js

```diff
@@ -63,5 +68,19 @@ export default class Kaiten extends Module {
 				</Fragment>,
 			}))
 		}
+
+		if (this._missedKaitenCasts > 0) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.HISSATSU_KAITEN.icon,
+				content: <Fragment>
+                                Always use <ActionLink {...ACTIONS.HISSATSU_KAITEN}/> on <ActionLink {...ACTIONS.MIDARE_SETSUGEKKA}/> and <ActionLink {...ACTIONS.HIGANBANA}/>. The gain on these actions from kaiten is too great to miss.
+				</Fragment>,
+				severity: SEVERITY.MAJOR,
```

Comment:
Should even one miss be `MAJOR`? I'd imagine you can probably step a medium -> major breakpoint in.